doc/abyss-pe.1

```diff
@@ -49,7 +49,7 @@ files containing single-end reads
 maximum number of branches of a bubble [2]
 .TP
 .B b
-maximum length of a bubble (bp) [10000]
+maximum length of a bubble (bp) [3*k]
```

Comment:
ABySS has two bubble popping stages. One in `ABYSS` and one in `PopBubbles`. The value of the abyss-pe `b` parameter is passed to [both stages](https://github.com/bcgsc/abyss/blob/master/bin/abyss-pe#L150-L153). The first stage handles only equal-length bubbles (no indels), does not check sequence similarity (except that the first and last `k-1 bp` must be identical, because de Bruijn graph), and [the default value is `3*k bp`](https://github.com/bcgsc/abyss/blob/master/Assembly/Options.cc#L346). The second stages handles arbitrary bubbles (indels too), requires a minimum sequence similarity of `p` ([default `0.9`](https://github.com/bcgsc/abyss/blob/master/bin/abyss-pe#L191)), and [the default value is `10000 bp`](https://github.com/bcgsc/abyss/blob/master/PopBubbles/PopBubbles.cpp#L99).

So, I think here it's more accurate to say that the maximum bubble length is `10000`, but it's complicated.